src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -201,7 +225,9 @@ describe( 'DefaultExporter', () => {
 
 		await exporter.export();
 
-		expect( fsPromises.unlink ).toHaveBeenCalledWith( '/tmp/studio_export_123/file.sql' );
+		expect( fsPromises.unlink ).toHaveBeenCalledWith(
+			'/tmp/studio_export_123/studio-backup-db-export-2023-07-31-12-00-00'
```

Comment:
Same as https://github.com/Automattic/studio/pull/399/files#r1700287847, the file should be exported with the `sql` extension.

Reply:
This is already solved as shared in https://github.com/Automattic/studio/pull/399/files#r1700304374.